### DISCUSSION TITLE
Key Binding Profile

### DISCUSSION THREAD

**Discussion by Deminosa** [ppy/osu#18080] [Ideas] [answered]
==[EN]==
I was wondering if it would be possible to create a key profile. For example, you could create a profile in Maina with which you can play with a keyboard and one with a controller. It always annoys me when I switch between my XBox 360 controller and keyboard, as well as vice versa, to reset the key'binding every time.

You could show up when starting the map, with in the GUI on the right, where you can already adjust the dimmability anyway.

What do you think?

==[DE]==
Ich habe mich gefragt, ob es möglich wäre ein Key-Profil zu erstellen. Somit könnte man in Maina zum Beispiel ein Profil erstellen, mit der man mit Tastatur Spiel und eins mit Controller. Mich nervt es immer, wenn ich zwischen meinem XBox 360 Controller und Tastatur wechsel, so wie um gekehrt, jedes Mal die Key'Bindung neu zu setzen.

Man könnte doch auf beim Starten der Map, mit im GUI rechts anzeigen, wo man ja eh schon die Dimmbarkeit einstellen kann.

Was meint ihr dazu?

**Comment by bdach** [comment]
If anything this would be supported by having more than 1 binding for the action, or something, rather than presets, I'd say.

**Reply by Deminosa** [reply]
So basically that wouldn't be possible, would it?

**Reply by Deminosa** [reply]
I mean that in the settings you say I want to have 2 profiles for Maina 4k. I call one controller and one keyboard. And in each of the profiles I set the key binds. And then choose a profile to play with.

**Reply by frenzibyte** [reply]
For your specific case, rather than creating a preset/profile for each input, you will be able to specify the keyboard key and the controller button that would activate each action. That'll also save the burden of having to switch the profile whenever switching between keyboard and controller.

**Comment by peppy** [comment, marked as answer]
We do plan on making full settings profiles. Which will include input settings, I believe.
